javascript/javascript1/week2/get-students.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+function addStudentToClass(studentName){
+    if(class07Students.length ===6 && studentName!=="Queen")
+       { return console.log(`Can not  add  more students to class 07`);
+    }
+    else if(class07Students.includes(studentName)){
+       return console.log(`Student ${studentName} is already in the class`); 
+    }
+    else if(studentName==="Queen"){
```

Comment:
Why do you need to check for the Queen twice... you already have a rule about the Queen in the beginning of the function.